versions.yml

```diff
@@ -185,35 +223,48 @@ modGroups:
       homepage: https://modrinth.com/mod/not-enough-animations
       modrinthId: MPCX6s5C
       desc: Adds missing animations from third-person to first-person
-      url: https://cdn.modrinth.com/data/MPCX6s5C/versions/WaI2x21x/notenoughanimations-fabric-1.7.4-mc1.21.jar
+      url: https://cdn.modrinth.com/data/MPCX6s5C/versions/EXt7Kt2t/notenoughanimations-fabric-1.7.6-mc1.21.jar
 
     - name: 3D Skin Layers
       license: tr7zw Protective License
       homepage: https://modrinth.com/mod/3dskinlayers
       modrinthId: zV5r3pPn
       desc: Creates 3d skin layers on skins
-      url: https://cdn.modrinth.com/data/zV5r3pPn/versions/8YK20yhu/skinlayers3d-fabric-1.6.6-mc1.21.jar
+      url: https://cdn.modrinth.com/data/zV5r3pPn/versions/h12GehRH/skinlayers3d-fabric-1.6.7-mc1.21.jar
 
     - name: Entity Texture Features
       license: LGPL 3.0 only
       homepage: https://modrinth.com/mod/entity-model-features
       modrinthId: 4I1XuqiY
-      desc: Adds Model Features to entities in Optifine compatible format.
-      url: https://cdn.modrinth.com/data/BVzZfTc1/versions/O3jDICoH/entity_texture_features_fabric_1.21-6.1.3.jar
+      desc: Adds Texture Features to entities in Optifine compatible format.
+      url: https://cdn.modrinth.com/data/BVzZfTc1/versions/2WL1sPyE/entity_texture_features_fabric_1.21-6.2.4.jar
 
     - name: Entity Model Features
       license: LGPL 3.0 only
       homepage: https://modrinth.com/mod/entitytexturefeatures
       modrinthId: BVzZfTc1
-      desc: Adds Texture Features to entities in Optifine compatible format.
-      url: https://cdn.modrinth.com/data/4I1XuqiY/versions/hX4kT2fu/entity_model_features_fabric_1.21-2.1.3.jar
+      desc: Adds Model Features to entities in Optifine compatible format.
+      url: https://cdn.modrinth.com/data/4I1XuqiY/versions/Qql6TI9W/entity_model_features_fabric_1.21-2.2.6.jar
 
     - name: Continuity
       license: LGPL-3
       homepage: https://modrinth.com/mod/continuity
       desc: Provides default connected textures
       url: https://cdn.modrinth.com/data/1IjD5062/versions/NksUpFjf/continuity-3.0.0-beta.5%2B1.21.jar
-      requires: [Indium]
+
+    - name: Barriers Don't Block Rain
+      license: CC0-1.0
+      homepage: https://modrinth.com/mod/barriers-dont-block-rain
+      modrinthId: nmDcB62a
+      desc: Allows weather effects to pass through barriers
+      url: https://cdn.modrinth.com/data/CXLh6wxz/versions/6tFgkvlI/barriers-dont-block-rain-1.0.5.jar
+
+    - name: Sodium Dynamic Lights
+      license: MIT
+      homepage: https://modrinth.com/mod/sodium-dynamic-lights
+      modrinthId: PxQSWIcD
+      desc: Dynamic Lights for Sodium
+      url: https://cdn.modrinth.com/data/PxQSWIcD/versions/67IEqds3/sodiumdynamiclights-fabric-1.0.2-1.21.1.jar
```

Comment:
Someone was having a crash with this mod yesterday, do you know if that's been fixed?

Reply:
I did not find any issues on my end, maybe they used outdated sodium? Tho I guess I will do more testing

Reply:
Pretty sure they couldn't even launch the game with the mod without it crashing so if you were able to get in then it must've been fixed

Reply:
That mod never got any updates yet, it is pretty new, so I suspect they just used outdated sodium or something like that